src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -26,6 +26,7 @@ import {
 import {requestExternalStoragePermission} from '../../../utils/permission';
 import {setChannel} from '../../../context/actions/setChannel';
 import {setParticipants} from '../../../context/actions/groupChat';
+import SignedMessageRepo from '../../../service/repo/signedMessageRepo';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `SignedMessageRepo` is added. It seems to be used for changing channel details later in the code. Ensure that all methods from this repository handle errors properly to avoid unexpected crashes.
```diff
+ import SignedMessageRepo from '../../../service/repo/signedMessageRepo';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->